start.bash

```diff
@@ -29,6 +36,25 @@ source $(dirname $0)/sqlflow/docker/dev/find_fastest_resources.sh
 # Find a way that we do not need to use 'set -e'
 set +e
 
+# Execute cmd until given output is present
+# "$1" is user message
+# "$2" is cmd
+# "$3" is expected output
+function wait_until() {
```

Comment:
How about polish this function to `repeate_check`, and do not use `exit` here, caller function can exit if necessary

``` bash
function repeated_check {
for I in {1...50}; then
    if [[ $("$cmd") == "$expected" ]]; then
        return 0
    else
         sleep 3
done
return 1
}
``` 

So the caller can use this function:
``` bash
Checking the API server is ready ...
if ! repeated_check "kubectl ...." "Running"; then
     echo "Failed and exit."
     exit 1
fi
```

Reply:
This will make every use of the function with multi-line code. Maybe we just keep it simple. In this script the function may just exit.